WME_Image_Overlays.user.js

```diff
@@ -5,39 +5,15 @@
-// @version     1.4.7
+// @version     1.4.8
 // @grant       none
 // ==/UserScript==
 
 /* global W, $, OpenLayers, I18n */
 
 var styleElement;
 
-function init(e) {
```

Comment:
Renamed function for simplicity for callBacks from even listeners.

Also, {e} is no longer needed as the new API allows for scripts to listen for events that occur when WME is in certain states. I made it so this script will not run unless the user is logged in and WME is fully ready.